fasttree/2.1.11/README.md

```diff
@@ -0,0 +1,80 @@
+Main tool: FastTree v2.1.11
+Full documentation: http://www.microbesonline.org/fasttree/
+
+**Docker container information:**
+
+This Docker container installs and runs FastTree using Ubuntu Bionic as a base image. No other dependencies are needed to run FastTree.
+The small sequence alignment (covid_alignment.fasta) also in the container is used to test the Docker build.
+
+**Usage for FastTree version 2.1.11**
+
+FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. 
+
```

Comment:
Could you include an example usage statement?

Something like the following should be sufficient
```
FastTree -log fasttree_test_log -nt sample.fasta > fasttree_test.nwk
```

Reply:
Will do!